pkg/asset/cluster/cluster.go

```diff
@@ -42,7 +44,7 @@ func (c *Cluster) Dependencies() []asset.Asset {
 func (c *Cluster) Generate(parents map[asset.Asset]*asset.State) (*asset.State, error) {
 	dir, err := terraform.BaseLocation()
 	if err != nil {
-		return nil, err
+		return nil, fmt.Errorf("error finding baselocation for terraform: %v", err)
```

Comment:
Do you have examples of what these errors look like on their own?  I'd rather not wrap them if they are already sufficiently clear.

Reply:
During debugging the error were not complete with context. this adds context.